config/hooks/normal/0510-install-ethercat-sources.hook.chroot

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+sudo mkdir -p /usr/local/share/keyrings/
+wget -O- https://build.opensuse.org/projects/science:EtherLab/signing_keys/download?kind=gpg | gpg --dearmor | sudo dd of=/usr/local/share/keyrings/science_EtherLab.gpg
+tee -a /etc/apt/sources.list.d/ighvh.sources > /dev/null <<EOT
```

Comment:
Why are you using 'tee' with /dev/null here?  It is cleaner to use
```
cat >> /etc/apt/sources.list.d/ighvh.sources <<EOT
```

Reply:
Using the recommended script supplied by IGh